<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableScreen</class>
 <widget class="QMainWindow" name="TableScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>400</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>711</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
